data/json/morale_types.json

```diff
@@ -135,7 +135,7 @@
     "text": "Ate Meat"
   },
   {
-    "id": "morale_meatarian",
```

Comment:
Don't change id! It will break all the saves plus all the code that might reference that mutation directly.